practice-one/src/index.html

```diff
@@ -3,44 +3,82 @@
-        <div>
-            <a href="#">
-                <img src="assets/images/photos/logo.png" alt=""/>
-            </a>
-        </div>
-        <nav class="header-section-navbar">
-            <ul class="navbar-menu">
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">HOME</a>                    
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark " href="#">PLACES</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">PRODUCTS</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">PROJECTS</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">PORTFOLIO</a>
-                </li>
-                <li>
-                    <a class="text-primary navbar-item text-dark" href="#">ABOUT US</a>
-                </li>
-            </ul>
-        </nav>
-    </header>
-    <!-- End Header -->
+
+    <!-- Start Footer -->
+    <footer class="footer-background">
+        <div class="footer-content container">
+            <div class="footer-list">
+                <div class="footer-item">
+                    <h4 class="item-title">contact us</h4>
+                    <address class="address">
+                        <p class="description">2300 W Arbutus Dr Citrus Springs, FL 34434, USA</p>
+                        <div>  
+                            <div>
+                                <a class="description" href="tel:+2357938917938">+235 793-891-7938</a>
+                            </div>
+                            <div>
+                                <a class="description" href="mailto:contact@hofmannui.net">contact@hofmannui.net</a>
+                            </div>
+                        </div>
+                    </address>
+                </div>
+                <div class="footer-item">
+                    <h4 class="item-title">follow us</h4>
+                    <div class="socials">
+                        <a href="#">
+                            <img class="social-icon" src="assets/images/icons/Pinterest.svg" alt="Pinterest"/>
+                        </a>
+                        <a href="#">
+                            <img class="social-icon" src="assets/images/icons/twitter.svg" alt="Twitter"/>
+                        </a>
+                        <a href="#">
+                            <img class="social-icon" src="assets/images/icons/facebook.svg" alt="Facebook"/>
+                        </a>
+                    </div>
+                </div>
+                <div class="footer-item">
+                    <h4 class="item-title">about us</h4>
+                    <p class="description">SmartWeb was made to help you quickly create beautiful<br>layouts of responsive websites.</p>
```

Comment:
Why do we need use tag br here?

Reply:
I use tag br here because in my desgin figma, this description is represented like this 
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/7c02efd9-aaf9-4054-abc8-3abdd387fa8f)
I see a break between create beautiful and layouts

Reply:
I fixed it, I used class limit-text instead used `<br>` tag

![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/8c567461-18ca-479a-be3d-6141408dd34a)